packages/manager/apps/key-management-service/src/pages/serviceKey/CreateKey.page.tsx

```diff
@@ -220,6 +222,44 @@ export default function CreateKey() {
           </div>
           <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-4 md:gap-6">
             <div className="flex flex-col gap-7 md:gap-9">
+              <div className="flex flex-col gap-6 md:gap-8">
+                <Subtitle>
+                  {t(
+                    'key_management_service_service-keys_create_protection_level_title',
+                  )}
+                </Subtitle>
+                <Description>
+                  {t(
+                    'key_management_service_service-keys_create_protection_level_subtitle',
+                  )}
+                </Description>
+                <OsdsTile
+                  rounded
+                  inline
+                  color={ODS_THEME_COLOR_INTENT.info}
+                  className="flex flex-col w-full h-fit"
+                >
+                  <div className="flex flex-col gap-6 pb-4 justify-center align-middle text-center">
+                    <CommonTitle>
+                      {t(
+                        'key_management_service_service-keys_create_software_protection_title',
+                      )}
+                    </CommonTitle>
+                    <Description>
+                      {t(
+                        'key_management_service_service-keys_create_software_protection_subtitle',
+                      )}
+                    </Description>
+
+                    <Description>
+                      <Subtitle>{KEY_SOFTWARE_PROTECTION_PRICE} €</Subtitle>{' '}
+                      {t(
+                        'key_management_service_service-keys_create_software_protection_price_suffix',
+                      )}
+                    </Description>
```

Comment:
You should use the `<Price />` component
packages/manager-react-components/src/components/content/price/price.component.tsx